packages/mds-policy-author/handlers/delete-policy.ts

```diff
@@ -26,6 +27,9 @@ export const DeletePolicyHandler = async (
 ) => {
   const { policy_id } = req.params
   try {
+    if (!res.locals.scopes.includes('policies:delete')) {
+      throw new AuthorizationError()
+    }
```

Comment:
I think the scope checking is normally done with middleware, when the handler is bound to the api instance?